tests/update/updates_packagekit_gpk.pm

```diff
@@ -14,13 +14,20 @@ use base "x11test";
 use strict;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_leap);
 
-sub turn_off_screensaver {
+sub turn_off_screensaver
+ {
+    my ($self) = @_;
     # Turn off screensaver
     x11_start_program('xterm');
     become_root;
     ensure_serialdev_permissions;
+    # Trying to detect bsc#1052258 which leads to the test failure occasionally
+    if(is_leap '42.3' && !script_run "dmesg | grep -P soft lockup - CPU#\\d+ stuck") {
```

Comment:
You need parentheses here, your conditoin is equal to:
```perl
if(is_leap ('42.3' && !script_run "dmesg....")) {
```
Which bubbles into `is_leap(0)` or `is_leap(1)`

Reply:
If a CPU is stuck I doubt you would be able to call the command but we might be lucky :)